package.json

```diff
@@ -48,11 +52,19 @@
     ]
   },
   "devDependencies": {
-    "ava": "^0.11.0",
-    "xo": "^0.12.1"
+    "ava": "^0.22.0",
```

Comment:
I would not have updated ava in this PR, that is about TypeScript.

The reason I did, is because this version of ava does include typings. So that's that.